subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/CompileOptions.java

```diff
@@ -505,4 +508,21 @@ public void setAnnotationProcessorGeneratedSourcesDirectory(File file) {
     public void setAnnotationProcessorGeneratedSourcesDirectory(Provider<File> file) {
         this.annotationProcessorGeneratedSourcesDirectory.set(file);
     }
+
+    /**
+     * Returns {@code true} if incremental annotation processing is enabled for this build.
```

Comment:
❌ remove "for this build". This is on `CompileOptions`, which can be different for every instance.

Reply:
Done.